### DISCUSSION TITLE
I use the version 3.2 is how to create the k8s component, set the mount directory, the host directory mount to the container, thank you

### DISCUSSION THREAD

**Discussion by PhilipWang34** [apache/dolphinscheduler#15561] [Q&A]
I use the version 3.2 is how to create the k8s component, set the mount directory, the host directory mount to the container, thank you

**Comment by Gallardot** [comment]
It is not supported in the current release, and there are no plans to support it at this time. [Apply policies using dynamic admission control](https://kubernetes.io/docs/concepts/policy/#apply-policies-using-dynamic-admission-control), You can dynamically mount pvc in a pod.

ref: https://github.com/apache/dolphinscheduler/issues/15368 https://github.com/apache/dolphinscheduler/pull/15369
